app/migrations.go

```diff
@@ -283,6 +284,32 @@ func (a *App) DoSystemConsoleRolesCreationMigration() {
 	}
 }
 
+func (a *App) doContentExtractionConfigMigration() {
+	// If the migration is already marked as completed, don't do it again.
+	if _, err := a.Srv().Store.System().GetByName(ContentExtractionConfigMigrationKey); err == nil {
+		return
+	}
+
+	if usersCount, err := a.Srv().Store.User().Count(model.UserCountOptions{}); err != nil {
+		mlog.Critical("Failed to get the users count for migrating the content extraction, using default value", mlog.Err(err))
+	} else {
+		if usersCount < 500 {
```

Comment:
Is this a hard limit we're setting right? Can we extract this number as a constant?

Reply:
Good point.

Reply:
done